src/test/java/com/fasterxml/jackson/databind/introspect/AnnotationIntrospectorPairTest.java

```diff
@@ -0,0 +1,103 @@
+package com.fasterxml.jackson.databind.introspect;
+
+import com.fasterxml.jackson.annotation.JacksonInject;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.OptBoolean;
+import com.fasterxml.jackson.databind.AnnotationIntrospector;
+import com.fasterxml.jackson.databind.BeanProperty;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.InjectableValues;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import junit.framework.TestCase;
+
+public class AnnotationIntrospectorPairTest extends TestCase {
```

Comment:
I think there is already `src/test/java/com/fasterxml/jackson/databind/introspect/IntrospectorPairTest.java` where this might fit?